benchmark-service/benchmark/utils.py

```diff
@@ -47,3 +47,15 @@ def get_data_as_string(self, file_path):
         except Exception:
             raise NameError()
         return data
+
+    def list_files_in_bucket(self, bucket_name):
+        list_of_files = []
+        bucket = self.storage_client.get_bucket(bucket_name)
+        for blob in bucket.list_blobs():
+            list_of_files.append('gs://' + bucket_name + '/' + blob.name)
+        return list_of_files
+
+    def list_files(self):
```

Comment:
Let's move this function out of the GoogleStorageReader class and it should be it's own utils function. We should have a list of valid benchmarks buckets that's a set of constants. You could either make a new file for that called `constants.py` or put it at the top of the `utils.py` file as `BENCHMARK_BUCKETS  = ['hail-benchmarks', 'hail-benchmarks-2']`. This function should call `list_files` for both of the buckets. So something like this:

```
def list_benchmark_files():
    list_of_files = []
    for bucket in BENCHMARK_BUCKETS:
       ...
```